Lesson33/Merve/src/components/Account/index.jsx

```diff
@@ -0,0 +1,30 @@
+import { UserDetailsForm } from '../UserDetailsForm';
+import { loggedInUser } from '../../data';
+import { useState } from 'react';
+
+export const Account = ({ completedTodos }) => {
```

Comment:
Account component should not know anything about completed todos. It's only used to update user information. 

Use PerformanceState component to show information about completed todos.

When username is changed in the UserDetailsForm component, you need to update the user state (created inside App.js).